event/v0.1.0/schema.json

```diff
@@ -0,0 +1,228 @@
+{       
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://common.schemas.verida.io/social/event/v0.1.0/schema.json",
+    "title": "Event",
+    "description": "Schema for a calendar event",
+    "type": "object",
+    "appearance": {
+        "style": {
+            "color": "#4285F4",
+            "icon": "./icon.svg"
+        }
+    },
+    "database": {
+        "name": "social_event",
+        "indexes": {
+            "startTime": [
+                "start.dateTime",
+                "start.date"
+            ],
+            "endTime": [
+                "end.dateTime",
+                "end.date"
+            ],
+            "organizer": [
+                "organizer.email"
+            ],
+            "attendees": [
+                "attendees.email"
+            ]
+        }
+    },
+    "layouts": {
+        "create": [
+            "summary",
+            "start",
+            "end",
+            "location"
+        ],
+        "view": [
+            "summary",
+            "description",
+            "start",
+            "end",
+            "location",
+            "organizer",
+            "attendees",
+            "status"
+        ]
+    },
+    "properties": {
+        "id": {
+            "title": "ID",
+            "description": "Unique identifier for the event",
+            "type": "string"
+        },
+        "status": {
+            "title": "Status",
+            "description": "Status of the event",
+            "type": "string",
+            "enum": [
+                "confirmed",
+                "tentative",
+                "cancelled"
+            ]
+        },
+        "created": {
+            "title": "Created",
+            "description": "Creation time of the event",
+            "type": "string",
+            "format": "date-time"
+        },
+        "updated": {
+            "title": "Updated",
+            "description": "Last modification time of the event",
+            "type": "string",
+            "format": "date-time"
+        },
+        "summary": {
+            "title": "Summary",
+            "description": "Title of the event",
+            "type": "string"
+        },
+        "description": {
+            "title": "Description",
+            "description": "Detailed description of the event",
+            "type": "string"
+        },
+        "location": {
+            "title": "Location",
+            "description": "Location of the event",
+            "type": "string"
+        },
+        "creator": {
+            "title": "Creator",
+            "description": "Details of the event creator",
+            "type": "object",
+            "properties": {
```

Comment:
Create a sub-schema for a "person" and use it here and also in the `organizer` section.